all-of-us/workbench
# [RW-1695][risk=low] Call out to FC for NIH linked username info

## Addresses 
[RW-1695](https://precisionmedicineinitiative.atlassian.net/browse/RW-1695)

This PR adds a new service call to FireCloud to get/store NIH information. It also updates the user's profile call to populate required fields in the user profile. I have not implemented any kind of update to the user's NIH information, that should be handled in [RW-1698](https://precisionmedicineinitiative.atlassian.net/browse/RW-1698)

## Testing
You can test this by:
* First examining the profile json returned from the profile call. It should normally show a null value for the linked nih username. 
* Next, log into https://firecloud.dsde-dev.broadinstitute.org/ with your aou account.
* Go to the profile page
* Follow the NIH login link - this takes you to a testing page that's not real, but does update FC dev.
* Enter `mbemis` into the username field and click the `Link to FireCloud` link
* Go back to AoU and refresh the profile page
* Examine the profile json and now the value should show `mbemis`
